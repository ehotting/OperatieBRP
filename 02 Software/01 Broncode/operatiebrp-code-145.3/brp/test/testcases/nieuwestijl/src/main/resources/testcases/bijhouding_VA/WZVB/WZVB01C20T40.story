Meta:
@status                 Klaar
@usecase                UCS-BY.0.VA

Narrative: R2326 Registratie verblijfsrecht niet toegestaan als de persoon wordt behandeld als Nederlander

Scenario: Registratie verblijfsrecht bij een persoon dat wordt behandeld als Nederlander indicatie gestart na peildatum
          LT: WZVB01C20T40

Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL-VA/LO3PL-WZVB/WZVB01C20T40-001.xls

When voer een bijhouding uit WZVB01C20T40.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Geslaagd
Then is het antwoordbericht gelijk aan /testcases/bijhouding_VA/WZVB/expected/WZVB01C20T40.xml voor expressie //brp:bhg_vbaRegistreerVerblijfsrecht_R
