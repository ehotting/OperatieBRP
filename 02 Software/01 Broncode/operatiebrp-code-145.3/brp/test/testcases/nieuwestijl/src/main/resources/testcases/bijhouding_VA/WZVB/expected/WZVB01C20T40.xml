<brp:bhg_vbaRegistreerVerblijfsrecht_R xmlns:brp="http://www.bzk.nl/brp/brp0200">
    <brp:stuurgegevens>
        <brp:zendendePartij>199901</brp:zendendePartij>
        <brp:zendendeSysteem>BRP</brp:zendendeSysteem>
        <brp:referentienummer>*</brp:referentienummer>
        <brp:crossReferentienummer>00000000-0000-0000-0042-200000000104</brp:crossReferentienummer>
        <brp:tijdstipVerzending>*</brp:tijdstipVerzending>
    </brp:stuurgegevens>
    <brp:resultaat>
        <brp:verwerking>Geslaagd</brp:verwerking>
        <brp:bijhouding>Verwerkt</brp:bijhouding>
        <brp:hoogsteMeldingsniveau>Geen</brp:hoogsteMeldingsniveau>
    </brp:resultaat>
    <brp:wijzigingVerblijfsrecht brp:objecttype="AdministratieveHandeling">
        <brp:partijCode>507013</brp:partijCode>
        <brp:tijdstipRegistratie>*</brp:tijdstipRegistratie>
        <brp:gedeblokkeerdeMeldingen>
            <brp:gedeblokkeerdeMelding brp:objecttype="GedeblokkeerdeMelding" brp:referentieID="comid.deblok.melding1">
                <brp:regelCode>R2493</brp:regelCode>
                <brp:melding>*</brp:melding>
            </brp:gedeblokkeerdeMelding>
        </brp:gedeblokkeerdeMeldingen>
        <brp:bijgehoudenPersonen>
            <brp:persoon brp:objecttype="Persoon">
                <brp:identificatienummers>
                    <brp:burgerservicenummer>807858249</brp:burgerservicenummer>
                </brp:identificatienummers>
            </brp:persoon>
        </brp:bijgehoudenPersonen>
        <brp:bijhoudingsplan brp:objecttype="Bijhoudingsplan">
            <brp:bijhoudingsvoorstelPartijCode>507013</brp:bijhoudingsvoorstelPartijCode>
            <brp:bijhoudingsplanPersonen>
                <brp:bijhoudingsplanPersoon brp:objecttype="BijhoudingsplanPersoon">
                    <brp:persoon brp:objecttype="Persoon">
                        <brp:afgeleidAdministratief>
                            <brp:tijdstipLaatsteWijziging>*</brp:tijdstipLaatsteWijziging>
                        </brp:afgeleidAdministratief>
                        <brp:identificatienummers>
                            <brp:burgerservicenummer>807858249</brp:burgerservicenummer>
                        </brp:identificatienummers>
                        <brp:bijhouding>
                            <brp:partijCode>507013</brp:partijCode>
                        </brp:bijhouding>
                    </brp:persoon>
                    <brp:situatieNaam>Indiener is bijhoudingspartij</brp:situatieNaam>
                </brp:bijhoudingsplanPersoon>
            </brp:bijhoudingsplanPersonen>
        </brp:bijhoudingsplan>
    </brp:wijzigingVerblijfsrecht>
</brp:bhg_vbaRegistreerVerblijfsrecht_R>
